src/components/Books.js

```diff
@@ -1,109 +1,11 @@
 import React from 'react';
-import styles from '../styles/Books.module.css';
+// import styles from '../styles/Books.module.css';
```

Comment:
- [ ] It is among the best practices to remove all the commented codes in your codebase. Please remove the above line of code to keep your codes clean 👍